src/System.Management.Automation/engine/PSVersionInfo.cs

```diff
@@ -271,6 +272,11 @@ internal static string FeatureVersionString
 
         internal static bool IsValidPSVersion(Version version)
         {
+            if (version.Major == s_psV7Version.Major)
+            {
+                return version.Minor == s_psV7Version.Minor;
+            }
+
             if (version.Major == s_psV6Version.Major)
```

Comment:
Should it be s_psV7Version?

Reply:
This is checking if it's compatible with v6 version